src/styles/header.scss

```diff
@@ -0,0 +1,78 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+
+  --main-color: #00acdc;
```

Comment:
It's recommended to define CSS variables (`--main-color`) in a root selector or a variables file for better maintainability, not within the `body` selector.